### PR TITLE
Fix broken member social links

### PR DESCRIPTION
**Pull request by kaylima**
The links were not working due to the combining of manually generated URLs + full URLs input by users in the Largo settings. This removes the first part of the link per issue #29.

### CODE REVIEW

File: single-inn_member.php [hunk 1]
```diff
@@ -33,23 +33,23 @@
 			<ul class="social">
 				<?php
 					if ( ! empty( $meta['_facebook_url'][0] ) ) {
-						$url = "https://fb.com/" . $meta['_facebook_url'][0];
+						$url = $meta['_facebook_url'][0];
```
Comment (rclations) [inline single-inn_member.php]: Can we add `esc_url()` around the urls - so like
```$url = esc_url( $meta['_facebook_url'][0] );```
https://codex.wordpress.org/Function_Reference/esc_url

**Comment by benlk** [comment]
@rclations does that application of esc_url look okay? esc_url_raw doesn't replace entities for display; esc_url does. I'm unsure which we should use in this case inside an attribute.

**Comment by rclations** [comment]
The example from the [codex](https://codex.wordpress.org/Function_Reference/esc_url_raw) seems to suggest that in this case we should be using `esc_url`

And actually, while we're at it, we don't really need to create a `$url` var to use it only once on the following line, so that can just be integrated into the echo statement

**Comment by benlk** [comment]
Alright, did that. One last check before merge?

**Pull request merged**